src/components/ContactUs/ContactUs.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+const ContactUs2 = () => {
```

Comment:
can you change this to `ContactUs`
```suggestion
const ContactUs = () => {
```